docs/eggs/installation.md

```diff
@@ -2,16 +2,7 @@
 
 To publish your first module to nest.land, you will need our CLI, **eggs**. You can install it using this command:
 ```shell script
-deno install -A -f --unstable -n eggs https://x.nest.land/eggs@0.2.3/mod.ts
-```
-
-Alternatively, to be informed of new releases you can install eggs with our global module manager:
-```shell script
-deno run -A --unstable https://x.nest.land/eggs@0.2.3/mod.ts install -A -f --unstable -n eggs https://x.nest.land/eggs@0.2.3/mod.ts
-```
-or
-```shell script
-eggs install -A -f --unstable -n eggs https://x.nest.land/eggs@0.2.3/mod.ts
+deno install -Afq --unstable https://x.nest.land/eggs@0.3.0/eggs.ts
 ```
 
 Please make sure to use the `-A` flag to grant all permissions to eggs, so you can enjoy all features seamlessly.
```

Comment:
```suggestion
Please make sure to use the `-Afq` flag to grant all permissions to eggs, so you can enjoy all features seamlessly.
```

Reply:
`-A` make more sense actually cuz `-f` and `-q` are not related to permissions